arkouda-env-dev.yml

```diff
@@ -0,0 +1,28 @@
+name: arkouda-env-dev
+channels:
+  - conda-forge
+dependencies:
+  - python>=3.7   # minimum 3.7
+  - numpy>=1.16.5,<=1.21.5
+  - pandas>=1.1.0
+  - pyzmq>=20.0.0
+  - tabulate
+  - pyfiglet
+  - versioneer
+  - matplotlib
+  - h5py
+  - pip
+  # Developer dependencies
+  - pexpect
+  - pytest
+  - pytest-env
+  - Sphinx

```

Comment:
This should be lowercase `sphinx`.

Reply:
this is supposed to be uppercase I think.
https://github.com/Bears-R-Us/arkouda/blob/b9e53938bce78a34be0b6f1631525f5b1db95f52/setup.py#L154-L156